website/config.toml

```diff
@@ -95,6 +95,10 @@ algolia_docsearch = true
 
 ##RELEASE_ADDME##
 
+[[params.versions]]
```

Comment:
since release branch is already there, is this required to be added again?
cc: @asmacdo

Reply:
I don't think we should be adding this as the release branch is already there and this is just a patch release.